.github/workflows/ghcr_runtime.yml

```diff
@@ -96,6 +117,19 @@ jobs:
           haskell: true
           large-packages: true
           swap-storage: true
+
+      # Forked repos can't push to GHCR, so we need to download the image as an artifact
+      - name: Download runtime image for fork
```

Comment:
You probably also need to add this Job for run integration tests?

Reply:
good catch!